2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,113 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+let favoriteRecipe = {
+  Title: "Jelly",
+  Serves: 3,
+  Ingredients: ["hot water", "cold water", "jelly", "sugar"],
+};
+
+console.log(favoriteRecipe.Title);
+console.log(`Serves: ${favoriteRecipe.Serves}`);
+console.log("Ingredients:");
+let ingred = favoriteRecipe.Ingredients;
+ingred.forEach((e) => console.log(e));
```

Comment:
Your submission fine. Please think of 2 ways to minimise code duplication?

Reply:
I'll refactor this code, and submit it again later.